myapp/app/controllers/tasks_controller.rb

```diff
@@ -2,10 +2,9 @@
 
 class TasksController < ApplicationController # rubocop:todo Style/Documentation
   def index
-    @tasks = Task.order(created_at: :asc)
-    return unless params[:sort] == 'latest'
+    @query = Task.ransack(params[:q])
 
-    @tasks = Task.order(created_at: :desc)
+    @tasks = @query.result(distinct: true).custom_order(params[:sort])
```

Comment:
do we need `distinct` option? 👀 is there any duplication? 🤔

Reply:
not yet though, but i think i should remove it now. i was thinking about future possibility